src/main/java/duke/commands/DeleteTaskCommand.java

```diff
@@ -0,0 +1,51 @@
+package duke.commands;
+
+import duke.exceptions.InvalidInputException;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a command to delete a task from the task list in the DailyTasks application.
+ * <p>
+ * This command deletes the task at the specified index and displays a message confirming the deletion.
+ * If the task index is invalid, no task will be deleted, and no message will be shown.
+ * </p>
+ */
+public class DeleteTaskCommand extends Command {
+
+    /** The index of the task to be deleted. */
+    private final int taskIndex;
+
+    /**
+     * Constructs a new DeleteTaskCommand with the specified task index.
+     *
+     * @param taskIndex The index of the task to be deleted. A value of -1 indicates an invalid index.
+     */
+    public DeleteTaskCommand(int taskIndex) {
+        super();
```

Comment:
No harm done, but I think there's no need for this call to the parent constructor